resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "closureDate": "2017-05-20"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: closeExistingLisaAccount
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      - $ref: '#/components/parameters/accountId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountRequest'
+            examples:
+              example-1:
+                value:
+                  accountClosureReason: All funds withdrawn
+                  closureDate: 2017-05-20
+              example-2:
+                value:
+                  accountClosureReason: Cancellation
+                  closureDate: 2017-05-20
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/lifetimeIsaManagerAccountsCloseAccountResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                badRequest:
+                  $ref: '#/components/examples/invalidLmrn'
+                invalidLmrn:
+                  $ref: '#/components/examples/invalidLmrn'
+                invalidAccountId:
+                  $ref: '#/components/examples/invalidAccountId'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                notOpenStatus:
+                  $ref: '#/components/examples/notOpenStatusUnauthorized'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                accountVoid:
+                  $ref: '#/components/examples/accountVoid'
+                accountClosed:
+                  $ref: '#/components/examples/accountClosed'
+                accountCancellationPeriodExceeded:
```

Comment:
Please add missing description

```
accountCancellationPeriodExceeded:
  description: You cannot close the account with cancellation as the reason because the cancellation period is over
  value:
    code: CANCELLATION_PERIOD_EXCEEDED
    message: You cannot close the account with cancellation as the reason because the cancellation period is over
```